project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.2.0")
```

Comment:
Can be upgraded to 3.4.0
<img width="749" alt="Screenshot 2021-08-11 at 15 32 06" src="https://user-images.githubusercontent.com/72032501/129058332-abe8b165-6362-4dc5-ac8a-e9c7ba250e7c.png">